model.py

```diff
@@ -8867,17 +8920,10 @@ class ExternalIntegration(Base):
         "ExternalIntegrationSetting", backref="external_integration",
         lazy="joined", cascade="save-update, merge, delete, delete-orphan",
     )
-
-    __table_args__ = (
-        UniqueConstraint('provider', 'type'),
-        Index(
-            'ix_unique_provider_with_null_type', provider,
-            unique=True, postgresql_where=(type.is_(None))),
-    )
-
```

Comment:
There are lots of situations -- not just with collections -- where different libraries in a single circ manager might need their own individual ExternalIntegrations for the same external integration. For example, two libraries on a circ manager might  have separate Novelist accounts.